src/module/dialog-flow.module.ts

```diff
@@ -18,10 +29,52 @@ export class DialogFlowModule implements NestModule {
 
 		return {
 			module: DialogFlowModule,
+			providers: [DialogFlowService],
 			controllers: [DialogFlowController.forRoot(webHookConfig)],
 		};
 	}
 
+	constructor(
+		private readonly modulesContainer: ModulesContainer,
+		private readonly moduleRef: ModuleRef, 
+		private readonly dialogFlowService: DialogFlowService,
+	) {}
+
+	public onModuleInit(): any {
```

Comment:
I suggest you to move this code into a separate file in the core directory as this example [https://github.com/adrien2p/nestjs-mandrill/blob/master/src/core/scanner.ts](https://github.com/adrien2p/nestjs-mandrill/blob/master/src/core/scanner.ts). It will be easier to test :D